packages/Material-Core.package/MDRenderer.class/class/translucentPixelsInRoundRectShapeBounds.radius.borderWidth..st

```diff
@@ -1,83 +1,83 @@
 drawing
```

Comment:
file too long. But I see it is hard to come by this problem, therefore this remains a suggestion :)